tests/cases/dom/traverse/next.js

```diff
@@ -0,0 +1,57 @@
+QUnit.module(
+    "next()",
+    {
+        beforeEach: function ()
+        {
+            document.getElementById("qunit-fixture").innerHTML =
+                '<div class="test-element-next element1"></div>' +
+                '<div class="test-element-next element2"></div>' +
+                'Some text' +
+                '<div class="test-element-next element3"></div>';
```

Comment:
Can we use 5+ elements and use one of the middle ones, so that we are sure that only the ones before/after are picked up?